hack/kind-cluster-build.sh

```diff
@@ -232,19 +245,19 @@ $HELM_BIN init --service-account=tiller --wait
 # FIXME: remove this
 docker pull gcr.io/google-containers/kube-scheduler:${k8sVersion}
 docker tag gcr.io/google-containers/kube-scheduler:${k8sVersion} mirantis/hypokube:final
-kind load docker-image --name=${clusterName} mirantis/hypokube:final
+$KIND_BIN load docker-image --name=${clusterName} mirantis/hypokube:final
```

Comment:
I guess this is not necessary right now. I'm fine to remove this in this PR or another PR.

Reply:
I'd like to remove it later because it requires additional test